fizzbuzz_playground/fizzbuzz.rb

```diff
@@ -0,0 +1,10 @@
+class FizzBuzz
+
+  def buzzoven num
+    s = ''
+    s += 'Fizz' if num % 3 == 0
+    s += 'Buzz' if num % 5 == 0
+    (s.empty? ? num : s)
```

Comment:
A habit I'm trying to overcome. Unfortunately by the time I finally break
myself of it we're just going to bring that and semi colons back in
Angular. And === and elseif. No!!!! :-)
On Mar 2, 2015 8:40 PM, "Brook Riggio" <notifications@github.com> wrote:

> In fizzbuzz_playground/fizzbuzz.rb
> <https://github.com/trendwithin/cf_exercises/pull/1#discussion_r25663025>:
>
> > @@ -0,0 +1,10 @@
> > +class FizzBuzz
> > +
> > +  def buzzoven num
> > +    s = ''
> > +    s += 'Fizz' if num % 3 == 0
> > +    s += 'Buzz' if num % 5 == 0
> > +    (s.empty? ? num : s)
>
> This method looks great, but... Why the parens on this line?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/trendwithin/cf_exercises/pull/1/files#r25663025>.
>